modules/installation-creating-aws-control-plane.adoc

```diff
@@ -176,6 +182,12 @@ and load balancing.
 . Copy the template from the *CloudFormation template for control plane machines*
 section of this topic and save it as a YAML file on your computer. This template
 describes the control plane machines that your cluster requires.
++
+[IMPORTANT]
+====
+If `m4` instance types are not available in your region, such as with
+`eu-west-3`, you must modify the CloudFormation template use `m5` types.
```

Comment:
If want to use `m5` type, have to update MasterInstanceType.AllowedValues list together to include it in https://docs.openshift.com/container-platform/4.1/installing/installing_aws_user_infra/installing-aws-user-infra.html#installation-cloudformation-control-plane_installing-aws-user-infra